scout/server/blueprints/panels/views.py

```diff
@@ -310,7 +327,7 @@ def panel_export_case_hits(panel_id):
             str(data["panel"]["version"]),
             institute_id,
             case_name,
-            datetime.datetime.now().strftime(DATETIME_FORMATTER),
+            datetime.datetime.now().strftime(DATE_DAY_FORMATTER),
```

Comment:
Why not just reuse the stored `now`?

Reply:
Which now? I wanted something that had just year month and day, not the whole thing with minutes seconds etc

Reply:
A few lines up..